src/main/java/org/apache/maven/plugins/dependency/tree/verbose/OsProperties.java

```diff
@@ -0,0 +1,91 @@
+package org.apache.maven.plugins.dependency.tree.verbose;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * Platform-dependent project properties normalized from ${os.name} and ${os.arch}. Netty uses these to select
+ * system-specific dependencies through the
+ * <a href='https://github.com/trustin/os-maven-plugin'>os-maven-plugin</a>.
+ */
+public class OsProperties
```

Comment:
does this need to be public? For that matter, is it used at all? I don't see where it's referenced.

Reply:
Yeah it's used in a static initialization block in VerboseDependencyGraphBuilder, I'll change it to package-private